src/js/components/Notification/Notification.js

```diff
@@ -0,0 +1,120 @@
+import React, { useContext } from 'react';
+import { ThemeContext } from 'styled-components';
+import { defaultProps } from '../../default-props';
+
+import { Layer } from '../Layer';
+import { Box } from '../Box';
+import { Button } from '../Button';
+import { Text } from '../Text';
+
+const Notification = ({ toast, message, body, status, onClose }) => {
+  const theme = useContext(ThemeContext) || defaultProps.theme;
+
+  const {
+    gap: containerGap,
+    elevation: containerElevation,
+    round: containerRound,
+  } = theme.notification.container;
+
+  const { pad: iconContainerPad, round: iconContainerRound } =
+    theme.notification.iconContainer;
+
+  const { pad: textContainerPad } = theme.notification.textContainer;
+
+  const { size: messageTextSize, weight: messageTextWeight } =
+    theme.notification.messageText;
+
+  const { size: bodyTextSize } = theme.notification.bodyText;
+
+  const { margin: buttonMargin } = theme.notification.button;
+
+  const Icons = {
+    StatusGood: theme.notification.icons.statusGood,
+    StatusWarning: theme.notification.icons.statusWarning,
+    StatusUnknown: theme.notification.icons.statusUnknown,
+    FormClose: theme.notification.icons.closeButton,
+  };
+
+  const iconColor = theme.notification.icons.color;
+
+  let icon;
+  let color;
+  switch (status) {
+    case 'warning':
+      color = theme.notification.status.warning.color;
+      icon = <Icons.StatusWarning color={iconColor} />;
+      break;
+    case 'good':
+      color = theme.notification.status.good.color;
+      icon = <Icons.StatusGood color={iconColor} />;
+      break;
+    case 'unknown':
+      color = theme.notification.status.unknown.color;
+      icon = <Icons.StatusUnknown color={iconColor} />;
+      break;
+    default:
+      color = theme.notification.status.unknown.color;
+      icon = <Icons.StatusUnknown color={iconColor} />;
+      break;
+  }
+
+  let content = (
+    <>
```

Comment:
I'm thinking this should be a `<Box direction="row" />` and I think this should be the `theme.notification.container` and make the one in the toast/Layer context something like `theme.notification.toast.container`

Reply:
I'm also thinking we can allow more theme variation by also including `{...theme.notification.<status>.container}`. That would allow setting a per-status background color for the whole notification.